src/index.html

```diff
@@ -22,6 +22,462 @@
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-quantity">Review: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-value">$ 2,199</p>
+        </div>
+
+        <div id="buy-button">
+          <a
+            class="card__button"
+            href="#buy-button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        />
+
+        <div class="card__description">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-quantity">Review: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-value">$ 2,199</p>
+        </div>
+
+        <div>
+          <a
+            class="card__button"
+            href="#buy-button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        />
+
+        <div class="card__description">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-quantity">Review: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-value">$ 2,199</p>
+        </div>
+
+        <div>
+          <a
+            class="card__button"
+            href="#buy-button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
```

Comment:
Similar to the previous comment, ensure that the 'buy' button's href attribute pointing to '#buy-button' is intentional. If different actions are intended, consider using unique IDs.